data_store/controllers/notify.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime
+
+from notifications_python_client.errors import HTTPError
+from notifications_python_client.notifications import NotificationsAPIClient
+from werkzeug.datastructures import FileStorage
+
+from config import Config
+from data_store.controllers.load_functions import get_or_generate_submission_id
+from data_store.db.queries import get_programme_by_id_and_round
+from submit.main.fund import FundConfig
+from submit.utils import get_friendly_fund_type
+
+
+def send_email(
+    email_address: str,
+    template_id: str,
+    notify_key: str | None,
+    personalisation: dict,
+) -> None:
+    """Send email to the specified email address via the GovUK Notify service.
+
+    :param email_address: recipient email address
+    :param template_id: template ID of GovNotify template
+    :param file: optional - file to send by email
+    :param personalisation: email personalisation values
+    :return: None
+    """
+
+    try:
+        notify_client = NotificationsAPIClient(notify_key)
+
+        notify_client.send_email_notification(
+            email_address=email_address,
+            template_id=template_id,
+            personalisation=personalisation,
+        )
+    except HTTPError as error:
```

Comment:
I am catching HTTPError.. despite not doing anything with it. Had second thoughts if it's worth it, but then I thought it might be nice to see what errors one should expect.

Reply:
I don't love this. HTTPError is one of many that could be thrown - don't think it makes much sense to call it out specifically. Error management in python is just one of those not-very-nice things.

Reply:
Should I remove the try-except then?

Reply:
yes please :)